server/server.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+
+var app = express();
+
+app.use(bodyParser.json());
+
+app.get('/card', (req, res) => {
```

Comment:
Maybe the route for the sender html should just be `'/'`. That way when the user went to our homepage, all they would have to type in would be `blahblahblah.com` instead of `blahblahblah.com/card`. What do you guys think?